<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: white;}</string>
   </property>
   <widget class="QFrame" name="loginframe">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>571</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#loginframe{
	background-color: rgb(244, 208, 185);
    border: 0px solid #555;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="emailBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#emailBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="QPushButton_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>550</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sansita</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #555;
    border-radius: 15px;
    border-style: outset;
    padding: 5px;
	color : black;
    background-color : rgb(136, 179, 160);
    }

QPushButton:hover {
	color : white;
    background-color : rgb(85, 170, 127)
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
     </property>
     <property name="text">
      <string>Daftar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Nama Lengkap</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="namaBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#namaBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#passwordBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Nomor Telepon</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telpBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#telpBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Alamat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adressBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#adressBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#usernameBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="posBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#posBox{
	background-color: white;
    border: 0px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	font: 12pt &quot;Sansita&quot;;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Sansita&quot;;</string>
     </property>
     <property name="text">
      <string>Kode Pos</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Sansita&quot;;</string>
    </property>
    <property name="text">
     <string>Form Registrasi</string>
    </property>
   </widget>
   <widget class="QFrame" name="navbar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#navbar{
	background-color: rgb(234, 216, 202);
   
    border-style: outset;
    padding: 5px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-30</y>
       <width>1211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame_2{
	background-color: rgb(136, 179, 160);
   
    border-style: outset;
    padding: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Logo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 26pt &quot;Sansita&quot;;
color : rgb(0, 0, 0) </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../../../Downloads/1E284E92-8854-46D1-BC93-31B6BD0FC54D-removebg-preview 1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="berandaButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sansita</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 0px solid #555;
    border-radius: 15px;
    border-style: outset;
    padding: 5px;
	color : rgb(53, 78, 59);
    background-color :  rgb(234, 216, 202);
    }

QPushButton:hover {
	color : white;
    }</string>
     </property>
     <property name="text">
      <string>Beranda</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="tanamanButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sansita</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 0px solid #555;
    border-radius: 15px;
    border-style: outset;
    padding: 5px;
	color :  rgb(53, 78, 59);
    background-color :  rgb(234, 216, 202);
    }

QPushButton:hover {
	color : white;
    }</string>
     </property>
     <property name="text">
      <string>Tanaman</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sansita</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 0px solid #555;
    border-radius: 15px;
    border-style: outset;
    padding: 5px;
	color : rgb(53, 78, 59);
    background-color :  rgb(234, 216, 202);
    }

QPushButton:hover {
	color : white;
    }</string>
     </property>
     <property name="text">
      <string>Tentang Kami</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../img/Search.png</normaloff>../../img/Search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="userButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>35</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../img/User.png</normaloff>../../img/User.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>41</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cartButton">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>30</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../img/Shopping Cart.png</normaloff>
       <normalon>../../img/Shopping Cart.png</normalon>../../img/Shopping Cart.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>31</width>
       <height>51</height>
      </size>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="notification">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../img/Doorbell.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
